pkg-manager/core/test/install/autoInstallPeers.ts

```diff
@@ -143,7 +143,7 @@ test('don\'t install the same missing peer dependency twice', async () => {
   const lockfile = await project.readLockfile()
   expect(Object.keys(lockfile.packages).sort()).toStrictEqual([
     '/@pnpm/y@1.0.0',
-    `/@pnpm.e2e/has-has-y-peer-peer@1.0.0${createPeersFolderSuffix([{ name: '@pnpm/y', version: '1.0.0' }, { name: '@pnpm.e2e/has-y-peer', version: '1.0.0' }])}`,
+    '/@pnpm.e2e/has-has-y-peer-peer@1.0.0(@pnpm.e2e/has-y-peer@1.0.0(@pnpm/y@1.0.0))(@pnpm/y@1.0.0)',
```

Comment:
the nested syntax here is pretty interesting—that's new, right? is it representing a small subtree of peer dep resolutions?

Reply:
Yes, if a peer dependency has peer dependencies of its own, they will be nested. The nesting may be deep if there is a long chain of peer dependency peer dependencies.